lib/testmap.py

```diff
@@ -236,7 +225,7 @@
 # their non-Atomic siblings.
 OSTREE_BUILD_IMAGE = {
     "fedora-coreos": "fedora-37",
-    "rhel4edge": "rhel-9-1",
+    "rhel4edge": "rhel-9-0",
```

Comment:
Is this desired that rhel4edge is updated down and not up?

Reply:
Yes, see the commit message. I tried to bump it to 9.2, but as the rhel4edge image itself is still 9.1, a 9.2 build [fails to install](https://cockpit-logs.us-east-1.linodeobjects.com/pull-4659-20230417-045930-0a83a092-rhel4edge-cockpit-project-cockpit/log.html). Upwards compatibility is not a problem.